pages/linux/xz.md

```diff
@@ -0,0 +1,30 @@
+# xz
+
+> Data compression software with a high compression ratio.
+> Can compress typical files ~30% smaller than gzip and ~15% smaller output than bzip2.
+> Syntax is similar to that of gzip.
```

Comment:
I have a doubt can pages with 4 lines of page description be created? Would there be any linter errors? @navarroaxel

Reply:
3 lines max

Reply:
I am not sure why tldr-bot didn't give an error. Wouldn't it recognise this?